server/master/src/main/java/org/apache/accumulo/master/Master.java

```diff
@@ -1498,51 +1498,47 @@ private void blockForTservers() throws InterruptedException {
           Property.MASTER_STARTUP_TSERVER_AVAIL_MAX_WAIT.getKey());
       maxWait = Long.MAX_VALUE;
     }
+    long sleepInterval = maxWait / 10;
 
-    // honor Retry condition that initial wait < max wait, otherwise use small value to allow thread
-    // yield to happen
-    long initialWait = Math.min(50, maxWait / 2);
-
-    Retry tserverRetry =
-        Retry.builder().infiniteRetries().retryAfter(initialWait, TimeUnit.MILLISECONDS)
-            .incrementBy(15_000, TimeUnit.MILLISECONDS).maxWait(maxWait, TimeUnit.MILLISECONDS)
-            .logInterval(30_000, TimeUnit.MILLISECONDS).createRetry();
+    // Set a incremental logging delay
+    long logIncrement = 15_000;
+    long logWait = 0, lastLog = 0;
```

Comment:
I think style conventions are that this should be two lines 
```
 long logWait = 0;
 long lastLog = 0;
```
but not sure if checksytle will complain - if not the it can be left as is.

Reply:
checkstyle didn't complain, but I switched it while I was in there.